robot/client/README.md

```diff
@@ -0,0 +1,57 @@
+# Go client
+The Go client for the Viam RDK can function as an SDK to connect to a robot.
+
+# Install
+
+To install the Go SDK, run
+
+	go get go.viam.com/rdk/robot/client
+
+# Basic Usage
+
+To connect to a robot as a client, you should instantiate a client.
+
+	package main
+
+	import (
+		"context"
+
+		"github.com/edaniels/golog"
+		"go.viam.com/rdk/robot/client"
+		"go.viam.com/rdk/utils"
+		"go.viam.com/utils/rpc"
+	)
+
+	func main() {
+		logger := golog.NewDevelopmentLogger("client")
+		robot, err := client.New(
```

Comment:
comments explaining most lines might be valuable

Reply:
added some more comments, but not sure at what level - feels like a lot of them should either be self explanatory or would be explaining go syntax instead